css-view-transitions-1/Overview.bs

```diff
@@ -787,6 +789,9 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	since that results in a circular dependency.
 	Therefore, the [=view transition layer=] is a sibling of all other content.
 
+	When a {{Document}}'s [=document/active view transition=]'s [=ViewTransition/phase=] is "`animating`",
+	any [=element=] in that {{Document}} that is [=captured in a view transition=] generates [=invisible=] boxes.
```

Comment:
the boxes generated by any element in that {{document}} that are [=captured in a view transition=] are [=invisible=].

Reply:
Done.